### PR TITLE
move coonfiguration from ecommerce_env file to yaml file

### PR DESCRIPTION
**Pull request by tehreem-sadat**

### CODE REVIEW

File: ecommerce/settings/devstack.py [hunk 1]
```diff
@@ -2,7 +2,10 @@
 # noinspection PyUnresolvedReferences
 from ecommerce.settings._debug_toolbar import *  # isort:skip
 from ecommerce.settings.production import *
-from os import environ
+import yaml
+
+with codecs.open(CONFIG_FILE, encoding='utf-8') as f:
```
Comment (asamolion) [inline ecommerce/settings/devstack.py]: You don't have to add this line to the `devstack.py` file. The yaml config is meant for production, not devstack. so please remove it.

File: ecommerce/settings/devstack.py [hunk 2]
```diff
@@ -48,8 +51,8 @@
...
         },
         'bluefin': {
-            'merchant_account_id': environ.get('BLUEFIN_ACCOUNT_ID'),
-            'api_access_key': environ.get('BLUEFIN_API_KEY'),
+            'merchant_account_id': config_from_yaml.get('BLUEFIN_ACCOUNT_ID'),
```
Comment (asamolion) [inline ecommerce/settings/devstack.py]: Similarly, these changes are not required in devstack.py. However, they are welcome in production.py

Comment (tehreem-sadat) [reply ecommerce/settings/devstack.py]: done

File: ecommerce/settings/devstack.py (hunk 1 above)
Comment (tehreem-sadat) [reply ecommerce/settings/devstack.py]: done

**Comment by tehreem-sadat** [comment]
@asamolion review this again please.

**Comment by asamolion** [comment]
Please merge and test extensively.

**Pull request merged**